plugin.js

```diff
@@ -64,7 +63,6 @@ function plugin (fastify, options, pluginRegistrationDone) {
       }
       if (!cached) {
         req.log.trace('session data missing (new/expired)')
-        req.session = getSession()
         return hookFinished()
       }
```

Comment:
Same thing as with the signature mismatch. We are creating a new empty session object here because the prior session expired.